models/public/densenet-121-cf2/densenet-121-cf2.md

```diff
@@ -0,0 +1,65 @@
+# densenet-121-cf2
```

Comment:
I think the suffix in the name should correspond to the identifier used in the `framework` field of the model field (i.e. `-caffe2`). Otherwise, we're just multiplying identifiers for no reason.

Reply:
`-cf2` suffix is used for consistency with `-tf`

Reply:
The difference with `tf` is that `tf` is also used as the `framework` field value, while `cf2` is not.